### PR TITLE
UCL-128 Adjust spacing for mobile resolutions

### PR DESCRIPTION
**Pull request by MichalGorskiOnegini**

### CODE REVIEW

File: src/Form/Wrapper/InputWrapper/InputWrapper.tsx [hunk 1]
```diff
@@ -95,6 +95,7 @@ const InputWrapperComponent: ForwardRefRenderFunction<HTMLDivElement, Props> = (
   const { prefix, suffix } = inputProps || {};
   const input = useRef<HTMLInputElement>(null);
   const hasValueOrActiveFloatingLabel = !!value || floatingLabelActive;
+  const helperIndent = window.innerWidth > 479 ? 20 : 16;
```
Comment (pawel-napieracz) [inline src/Form/Wrapper/InputWrapper/InputWrapper.tsx:98]: you need to take a font-size from document.body and calculate px as browsers does with rems.

File: src/Form/Wrapper/SelectWrapper/SelectWrapper.module.scss [hunk 2]
```diff
@@ -22,9 +22,16 @@
...
+  margin-left: var(--form-element-horizontal-padding-mobile);
 }
 
 .floating-label-active [data-display] {
   top: calc(50% + 0.5rem);
 }
+
+@media only screen and (min-width: 480px) {
+  .select-helper-text {
+    margin-top: 0.25rem;
```
Comment (pawel-napieracz) [inline src/Form/Wrapper/SelectWrapper/SelectWrapper.module.scss:34]: margin-top is not needed

File: stories/Form/Form.stories.tsx [hunk 3]
```diff
@@ -330,3 +330,7 @@ const Template: Story<Props> = args => {
 export const Form = Template.bind({});
 
 Form.args = {};
+
+Form.parameters = {
+  chromatic: { viewports: [400, 1200] }
```
Comment (pawel-napieracz) [inline stories/Form/Form.stories.tsx:335]: I'm glad you did it only for one story :). Coole we have one story that has all form elements

**Review by pawel-napieracz — CHANGES_REQUESTED** [review]

File: src/Form/Fieldset/Fieldset.module.scss [hunk 4]
```diff
@@ -16,7 +16,7 @@
 
 .fieldset {
   border: 0;
-  padding: 1rem 1.25rem;
+  padding: 1rem var(--form-element-horizontal-padding-mobile);
```
[the remark this answers is not in the document]
Comment (MichalGorskiOnegini) [reply src/Form/Fieldset/Fieldset.module.scss:19]: you are correct, my bad :D

File: stories/Form/Form.stories.tsx (hunk 3 above)
Comment (MichalGorskiOnegini) [reply stories/Form/Form.stories.tsx:335]: indeed ;)

File: src/Form/Wrapper/SelectWrapper/SelectWrapper.module.scss (hunk 2 above)
Comment (MichalGorskiOnegini) [reply src/Form/Wrapper/SelectWrapper/SelectWrapper.module.scss:34]: oops

File: src/Form/Wrapper/InputWrapper/InputWrapper.tsx (hunk 1 above)
Comment (MichalGorskiOnegini) [reply src/Form/Wrapper/InputWrapper/InputWrapper.tsx]: done

File: src/Form/Wrapper/InputWrapper/InputWrapper.tsx [hunk 5]
```diff
@@ -95,7 +96,7 @@ const InputWrapperComponent: ForwardRefRenderFunction<HTMLDivElement, Props> = (
   const { prefix, suffix } = inputProps || {};
   const input = useRef<HTMLInputElement>(null);
   const hasValueOrActiveFloatingLabel = !!value || floatingLabelActive;
-  const helperIndent = window.innerWidth > 479 ? 20 : 16;
+  const helperIndent = window.innerWidth > remToPx(30) ? remToPx(1.25) : remToPx(1);
```
[the remark this answers is not in the document]
Comment (pawel-napieracz) [reply src/Form/Wrapper/InputWrapper/InputWrapper.tsx]: fixed

**Review by pawel-napieracz — APPROVED** [review]

**Pull request merged**